### PR TITLE
Add --run-immediately options

### PR DESCRIPTION
**Pull request by tamago324**
Could you tell me if anything is wrong with the option name or help text?

### CODE REVIEW

File: autoload/leaderf/python/leaderf/commandExpl.py [hunk 1]
```diff
@@ -73,16 +75,20 @@ def _defineMaps(self):
...
-            lfCmd("call histadd(':', '%s')" % escQuote(cmd))
-        except vim.error as e:
-            error = lfEncode(str(repr(e)))
-            if "E471" in error:
-                # Arguments mandatory
-                lfCmd("call feedkeys(':%s ')" % escQuote(cmd))
-            else:
-                lfPrintError(e)
+
+        if self._getExplorer()._run_immediately:
```
Comment (Yggdroot) [inline autoload/leaderf/python/leaderf/commandExpl.py]: Can be replaced by 
`if "--run-immediately" in self._arguments:`
So self._run_immediately can be removed from CommandExplorer.

File: autoload/leaderf/python/leaderf/commandExpl.py [hunk 2]
```diff
@@ -73,16 +75,20 @@ def _defineMaps(self):
...
+            try:
+                lfCmd(cmd)
+                lfCmd("call histadd(':', '%s')" % escQuote(cmd))
+            except vim.error as e:
+                error = lfEncode(str(repr(e)))
+                if "E471" in error:
+                    # Arguments mandatory
+                    lfCmd("call feedkeys(':%s ')" % escQuote(cmd))
```
Comment (Yggdroot) [inline autoload/leaderf/python/leaderf/commandExpl.py]: I think it is better to use `feedkeys(':%s ', 'n')`, i.e., Do not remap keys.

File: autoload/leaderf/python/leaderf/commandExpl.py (hunk 1 above)
Comment (tamago324) [reply autoload/leaderf/python/leaderf/commandExpl.py]: I didn't know there were a `self._arguments`.
Thanks!

File: autoload/leaderf/python/leaderf/commandExpl.py (hunk 2 above)
Comment (tamago324) [reply autoload/leaderf/python/leaderf/commandExpl.py]: > Do not remap keys.

That looks better!
I fixed.

Comment (Yggdroot) [reply autoload/leaderf/python/leaderf/commandExpl.py]: could u also modify here？
https://github.com/Yggdroot/LeaderF/blob/614a2fe042d1df9d682114a48335fa5387a4ae36/autoload/leaderf/python/leaderf/historyExpl.py#L136

Comment (tamago324) [reply autoload/leaderf/python/leaderf/commandExpl.py]: OK.
I fixed.

**Comment by tamago324** [comment]
Thank you!

**Pull request merged**